fdbclient/SystemData.cpp

```diff
@@ -1627,11 +1627,6 @@ BlobWorkerInterface decodeBlobWorkerListValue(ValueRef const& value) {
 	return interf;
 }
 
-const KeyRangeRef tenantMapKeys("\xff/tenant/map/"_sr, "\xff/tenant/map0"_sr);
-const KeyRef tenantMapPrefix = tenantMapKeys.begin;
-const KeyRef tenantMapPrivatePrefix = "\xff\xff/tenant/map/"_sr;
-const KeyRef tenantLastIdKey = "\xff/tenant/lastId"_sr;
```

Comment:
You should remove these keys from `SystemData.h` has well

Reply:
These were removed in SystemData.h in a [previous commit](https://github.com/apple/foundationdb/commit/17146c484b53ce25b221de532d8a225c1aa805a5#diff-27a91ad0e4fc1fbbb9d16a98c09a701692a82f71f361dda35c4679a1e80731d6), but I forgot to do so here. These are currently unused and I guess I included cleaning them up in this PR.